yjit.rb

```diff
@@ -139,22 +139,9 @@ def self.runtime_stats
     side_exits = total_exit_count(stats)
     total_exits = side_exits + stats[:leave_interp_return]
 
-    # Number of instructions that finish executing in YJIT.
-    # See :count-placement: about the subtraction.
-    retired_in_yjit = stats[:exec_instruction] - side_exits
-
-    # Average length of instruction sequences executed by YJIT
-    avg_len_in_yjit = retired_in_yjit.to_f / total_exits
-
-    # Proportion of instructions that retire in YJIT
-    total_insns_count = retired_in_yjit + stats[:vm_insns_count]
-    yjit_ratio_pct = 100.0 * retired_in_yjit.to_f / total_insns_count
```

Comment:
I think you can compute everything except the interpreter instruction count and the ratio in YJIT outside of a stats build.

IMO the average length in YJIT is still a useful stat to produce and compare, and the number of instructions executed by YJIT is too.

Reply:
Cool. I'll make those changes and fix the issue @k0kubun mentioned.